styles.css

```diff
@@ -0,0 +1,130 @@
+body {
+  min-height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.container {
+  height: 500px;
+  background: white;
+  width: 375px;
+  border: 1px solid #ccc;
+  box-shadow: 0 4px 10px 0 rgba(204, 77, 77, 0.2);
```

Comment:
- [ ] [ OPTIONAL] I suggest you remove the border to your container so  that your home page will not look take the card as shown in the image below :+1:  :grinning:  :grinning:  :grinning: 

## your design 

![image](https://user-images.githubusercontent.com/95297251/191931658-085e824a-c9ec-4c03-b706-f5b217b11be3.png)

## Figma design
![image](https://user-images.githubusercontent.com/95297251/191932059-6f8777a7-9bea-4bff-b992-efd0503aeda5.png)